src/io/flutter/run/daemon/FlutterApp.java

```diff
@@ -522,6 +520,7 @@ public String deviceId() {
 
   public void setFlutterDebugProcess(FlutterDebugProcess flutterDebugProcess) {
     myFlutterDebugProcess = flutterDebugProcess;
+    myFlutterLog.setFlutterDebugProcess(flutterDebugProcess);
```

Comment:
I wonder if it makes sense to have the FlutterLog get the debug process off the FlutterApp? it isn't great that the FlutterApp is the grab bag of all Flutter state but at least that means we only have to keep track of the state in one place.

Reply:
Maybe.  This definitely could benefit from some general cleaning up.  (On my list! ☑️ )